cmd/util.go

```diff
@@ -90,65 +90,39 @@ func verifyBin(bin string, psFunc func(string) string) bool {
 	return strings.Contains(out, bin)
 }
 
-func verifyKubeVersion(major string, minor string) {
-	// These executables might not be on the user's path.
+type version struct {
+	Server string
+	Client string
+}
 
+func getKubeVersion() *version {
+	ver := new(version)
+	// These executables might not be on the user's path.
 	_, err := exec.LookPath("kubectl")
 	if err != nil {
-		continueWithError(err, sprintlnWarn("Kubernetes version check skipped"))
-		return
+		s := fmt.Sprintf("Kubernetes version check skipped with error %v", err)
+		continueWithError(err, sprintlnWarn(s))
+		return nil
 	}
 
 	cmd := exec.Command("kubectl", "version")
 	out, err := cmd.Output()
 	if err != nil {
-		s := fmt.Sprintf("Kubernetes version check skipped with error %v", err)
+		s := fmt.Sprintf("Kubernetes version check skipped, with error getting kubectl version")
 		continueWithError(err, sprintlnWarn(s))
-		if len(out) == 0 {
-			return
-		}
-	}
-
-	msg := checkVersion("Client", string(out), major, minor)
-	if msg != "" {
-		continueWithError(fmt.Errorf(msg), msg)
-	}
-
-	msg = checkVersion("Server", string(out), major, minor)
-	if msg != "" {
-		continueWithError(fmt.Errorf(msg), msg)
+		return nil
```

Comment:
The test for len(out) == 0 allowed for the case where kubectl gives client version but not server.  We get an error but there is still have some kubectl output to check so we can give the user some info about what version is running. So I think we should add it back in.